docs/src/modules/components/BaseUIComponents.js

```diff
@@ -0,0 +1,182 @@
+      href: '/base/react-input/',
+    },
+    {
+      title: 'Menu',
+      srcLight: '/static/images/base-ui-components/menu-light.png',
+      srcDark: '/static/images/base-ui-components/menu-dark.png',
+      href: '/base/react-menu/',
+    },
+    {
+      title: 'Modal',
+      srcLight: '/static/images/base-ui-components/modal-light.png',
+      srcDark: '/static/images/base-ui-components/modal-dark.png',
+      href: '/base/react-modal/',
+    },
+    {
+      title: 'No-SSR',
+      srcLight: '/static/images/base-ui-components/no-ssr-light.png',
+      srcDark: '/static/images/base-ui-components/no-ssr-dark.png',
+      href: '/base/react-no-ssr/',
+    },
+    {
+      title: 'Popper',
+      srcLight: '/static/images/base-ui-components/popper-light.png',
+      srcDark: '/static/images/base-ui-components/popper-dark.png',
+      href: '/base/react-popper/',
+    },
+    {
+      title: 'Portal',
+      srcLight: '/static/images/base-ui-components/portal-light.png',
+      srcDark: '/static/images/base-ui-components/portal-dark.png',
+      href: '/base/react-portal/',
+    },
+    {
+      title: 'Select',
+      srcLight: '/static/images/base-ui-components/select-light.png',
+      srcDark: '/static/images/base-ui-components/select-dark.png',
+      href: '/base/react-select/',
+    },
+    {
+      title: 'Slider',
+      srcLight: '/static/images/base-ui-components/slider-light.png',
+      srcDark: '/static/images/base-ui-components/slider-dark.png',
+      href: '/base/react-slider/',
+    },
+    {
+      title: 'Snackbar',
+      srcLight: '/static/images/base-ui-components/snackbar-light.png',
+      srcDark: '/static/images/base-ui-components/snackbar-dark.png',
+      href: '/base/react-snackbar/',
+    },
+    {
+      title: 'Switch',
+      srcLight: '/static/images/base-ui-components/switch-light.png',
+      srcDark: '/static/images/base-ui-components/switch-dark.png',
+      href: '/base/react-switch/',
+    },
+    {
+      title: 'Table Pagination',
+      srcLight: '/static/images/base-ui-components/pagination-light.png',
+      srcDark: '/static/images/base-ui-components/pagination-dark.png',
+      href: '/base/react-table-pagination/',
+    },
+    {
+      title: 'Tabs',
+      srcLight: '/static/images/base-ui-components/tabs-light.png',
+      srcDark: '/static/images/base-ui-components/tabs-dark.png',
+      href: '/base/react-tabs/',
+    },
+    {
+      title: 'Textarea Autosize',
+      srcLight: '/static/images/base-ui-components/textarea-light.png',
+      srcDark: '/static/images/base-ui-components/textarea-dark.png',
+      href: '/base/react-textarea-autosize/',
+    },
+  ];
+}
+
+function BaseUIComponents() {
+  return (
+    <Grid container spacing={2} sx={{ pt: 2, pb: 4 }}>
+      {components().map((component) => (
+        <Grid item xs={12} sm={4} sx={{ flexGrow: 1 }} key={component.title}>
+          <Card
+            component="a"
```

Comment:
```suggestion
            component={Link}
```

The `Link` should come from `import Link from 'docs/src/modules/components/Link';`. NextJS will handle to preload and prevent full page reload when you click on the link.

Reply:
Oh, just saw the target="_blank".